sdks/java/io/google-cloud-platform/src/main/java/org/apache/beam/sdk/io/gcp/datastore/DatastoreV1.java

```diff
@@ -1447,9 +1471,20 @@ private void flushBatch() throws DatastoreException, IOException, InterruptedExc
           continue;
         }
 
+        HashMap<String, String> baseLabels = new HashMap<>();
+        baseLabels.put(MonitoringInfoConstants.Labels.PTRANSFORM, "");
+        baseLabels.put(MonitoringInfoConstants.Labels.SERVICE, "Datastore");
+        baseLabels.put(MonitoringInfoConstants.Labels.METHOD, "flushBatch");
```

Comment:
Lets use this method name
"BatchDatastoreWrite"

To refer to writing a batch of elements to datastore.
Thus, if we change the API in a refactoring at some point, we can continue to use this method label.

Reply:
Change done